### PR TITLE
elegant solution for k8s 1.18

### PR DESCRIPTION
**Pull request by k1rk**
only till https://github.com/kubernetes-sigs/controller-tools/issues/444 be resolved

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,10 +45,19 @@ install-develop-tools: install-golint
...
 
-gen:
+gen-crd:
 	$(OPERATOR_BIN) generate crds --crd-version=v1beta1
 	$(OPERATOR_BIN) generate k8s
 
+fix118:
+	docker run --rm -v "${PWD}":/workdir mikefarah/yq /bin/sh -c " \
+		$(YAML_MERGE) deploy/crds/victoriametrics.com_vmagents_crd.yaml scripts/patch1.18.yaml && \
+		$(YAML_MERGE) deploy/crds/victoriametrics.com_vmalertmanagers_crd.yaml scripts/patch1.18.yaml && \
```
Comment (f41gh7) [inline Makefile]: yq d -i deploy/crds/victoriametrics.com_vmagents_crd.yaml spec.validation.openAPIV3Schema.properties.spec.properties..containers.items.properties

@k1rk  what do you think? we can simply  remove properties for this crd

Comment (k1rk) [reply Makefile]: added

Comment (k1rk) [reply Makefile]: or actually removed :D

Comment (f41gh7) [reply Makefile]: ah, i mean to apply it as general solution for all containers and initContainers defenitions :)

Comment (k1rk) [reply Makefile]: 👌  done

**Pull request merged**